selfdrive/loggerd/logger.cc

```diff
@@ -253,7 +253,8 @@ int logger_next(LoggerState *s, const char* root_path,
   pthread_mutex_unlock(&s->lock);
 
   // write beggining of log metadata
-  log_init_data(s);
+  auto bytes = s->init_data.asBytes();
```

Comment:
if we save the bytes, we don't have to do this conversion every time

Reply:
The kj library has no obvious way to do this,so I used std::vector to make a copy

Reply:
Maybe the original version will be better.adding a conversion will not have any impact.

Reply:
There is a function `releaseAsBytes` can be used

Reply:
sorry,I'm wrong,we can't call releaseAsBytes on kj::Array<capnp::word>:
` error: static_assert failed due to requirement 'sizeof(capnp::word) == sizeof(unsigned char)' "releaseAsBytes() only possible on arrays with byte-size elements (e.g. chars)."`